cmd/osd-sanity/sanity_test.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2017 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package sanity
+
+import (
+	"flag"
+	"fmt"
+	"testing"
+
+	"github.com/libopenstorage/openstorage/pkg/sanity"
+)
+
+const (
+	prefix string = "osd."
+)
+
+var (
+	VERSION  = "(dev)"
```

Comment:
this could be a constant, right?

Reply:
Always used a var. I can check if it needs to be a constant. The whole point is that it needs to be overriden by the linker to apply the _real_ version.

Reply:
Just checked. cannot be a const, must be a var.